core/primitives-core/Cargo.toml

```diff
@@ -21,6 +21,7 @@ enum-map.workspace = true
 num-rational.workspace = true
 serde.workspace = true
 serde_repr.workspace = true
+serde_with.workspace = true
```

Comment:
Can we re-export it in `near-sdk` as well, similar to how it was done [here](https://github.com/mitinarseny/near-sdk-rs/blob/327ebe55e97e0f3f168679d0ed5b733a13658581/near-sdk-macros/src/lib.rs#L180-L189)? 🙏🏻

Reply:
I don't see why we couldn't do that in the near-sdk. But this is a different repo maintained by different people. :)

Do we need to do anything here in near-primitives-core?